plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -31,174 +30,185 @@
 from .figure_resampler_interface import AbstractFigureAggregator
 from .utils import is_figure, is_fr
 
+try:
```

Comment:
Just a suggestion, maybe we can move the persistent inline output to a new file? (What are your opinions? @NielsPraet @jvdd)

Reply:
Yes that would clean up the code just a bit, so I think this is a great idea

Reply:
👍🏼